code/game/machinery/exonet_node.dm

```diff
@@ -1,6 +1,6 @@
 /obj/machinery/exonet_node
 	name = "exonet node"
-	desc = "This machine is one of many, many nodes inside Vir's section of the Exonet, connecting the Northern Star to the rest of the system, at least \
+	desc = "This machine is one of many, many nodes inside Vir's section of the Exonet, connecting Virgo Station to the rest of the system, at least \
```

Comment:
This is incorrect for a number of reasons.

Vir is the name of the Northern Star. That should be changed to Erigone.

Furthermore, "Virgo Station" should be replaced with [station_name] or NSS Adephagia (our official moniker that no one but CentCom uses).